checker/tests/index/LessThanFloat.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.checker.index.qual.LessThan;
+
+public class LessThanFloat {
+    int bigger;
+
+    @LessThan("bigger") byte b;
+
+    @LessThan("bigger") short s;
+
+    @LessThan("bigger") int i;
+
+    @LessThan("bigger") long l;
+
+    // :: error: (anno.on.nonintegral)
+    @LessThan("bigger") float f;
+
+    // :: error: (anno.on.nonintegral)
+    @LessThan("bigger") double d;
+
+    // :: error: (anno.on.nonintegral)
+    @LessThan("bigger") boolean bool;
+
+    @LessThan("bigger") char c;
```

Comment:
Can you add tests for the primitive wrapper types (e.g. `java.lang.Integer`) here, as well? I think the code as-is handles them correctly, but it would be good to be sure.

Reply:
The code did *not* handle primitive wrapper types.  (There was a todo comment about that.)  I have enhanced the code to do so, removed the todo, and added tests.